TRENZ.Lib.RazorMail.Core/Models/MailHeaderCollection.cs

```diff
@@ -165,6 +188,13 @@ public void AddBlindCarbonCopy(IEnumerable<MailAddress> addresses) =>
         .ExceptBy(AddressKeys, x => x.Key)
         .ToDictionary(x => x.Key, x => x.Value);
 
+    /// <summary>
+    /// Gets the headers that the <see cref="BaseSmtpMailClient"/> implementation should just iterate over.
+    /// </summary>
+    public IReadOnlyDictionary<string, object> NonSpecificHandledHeaders => this
```

Comment:
Do we need `NonAddressHeaders` anymore?

Trying to keep the exposed API minimal, I'd opt to at least mark it obsolete, if not remove it entirely

Reply:
Perhaps not. I'll leave that up to you.

Reply:
Make it obsolete, we'll remove it in the next version

Reply:
Hm. It's still used in `AppendRange`. Will we inline that instead, then? Or make it private?

Reply:
Hmmm... I'd say make it private